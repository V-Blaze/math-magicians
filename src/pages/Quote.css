.quote-main {
  padding: 20% 15%;
  display: flex;
  justify-content: center;
  align-items: center;
}
