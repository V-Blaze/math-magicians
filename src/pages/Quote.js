import React from 'react';

// stylesheet
import './Quote.css';

const Quote = () => (
  <div className="quote-main">
    <h3>
      I have seen the excitement and energy toward math increase in a profound way with Prodigy.
      The kids are inspired. They work together, they collaborate and help one another.
      â€” Sue Tresselt,
    </h3>
  </div>
);

export default Quote;
