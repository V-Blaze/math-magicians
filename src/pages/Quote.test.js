import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import React from 'react';
import Quote from './Quote';

it('snapshots', () => {
  const tree = renderer
    .create(<Quote />).toJSON();
  expect(tree).toMatchSnapshot();
});
it('testing the heading inside Quote component', () => {
  render(<Quote />);
  const heading = screen.getByText(/The kids are inspired. They work together, they collaborate and help one another./i);
  expect(heading).toBeInTheDocument();
});
