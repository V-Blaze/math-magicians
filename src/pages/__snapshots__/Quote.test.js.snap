// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshots 1`] = `
<div
  className="quote-main"
>
  <h3>
    I have seen the excitement and energy toward math increase in a profound way with Prodigy. The kids are inspired. They work together, they collaborate and help one another. â€” Sue Tresselt,
  </h3>
</div>
`;
